doc/source/reference/security.rst

```diff
@@ -0,0 +1,62 @@
+NumPy security considertions and reporting
+==========================================
+
+Security issues can be reported privately as described on the issue tracker.
+The [Python security reporting guidlines](https://www.python.org/dev/security/)
+are a good resource and its notes apply also to NumPy.
+
+NumPy's maintainers are not security experts.  However, we are conscientious
+about security and experts of both the code NumPy and its typical use.
+Please do notify us before creating security advisories against NumPy as
+we are happy to prioritize issues or help with assessing the severity of a bug.
+A security advisory we are not aware of beforehand can lead to a lot of work
+for all involved parties.
+
+
+Advise for using NumPy on untrusted data
+----------------------------------------
+
+A user who can freely execute NumPy (or Python) functions must be considered
+to have the same privilege as the process/Python interpreter.
+If an untrusted user has direct access to parts of the NumPy API,
+a the whole process must generally be isolated.
```

Comment:
typo?

Reply:
Yeah, but removed the whole sentence, it did not really seem to add much.